dll/win32/cryptui/main.c

```diff
@@ -789,11 +789,15 @@ static LRESULT CALLBACK cert_mgr_advanced_dlg_proc(HWND hwnd, UINT msg,
         {
         case IDOK:
             save_cert_mgr_usages(hwnd);
+#ifndef __REACTOS__
             ImageList_Destroy((HIMAGELIST)GetWindowLongPtrW(hwnd, DWLP_USER));
+#endif
             EndDialog(hwnd, IDOK);
             break;
         case IDCANCEL:
+#ifndef __REACTOS__
             ImageList_Destroy((HIMAGELIST)GetWindowLongPtrW(hwnd, DWLP_USER));
+#endif
```

Comment:
This imagelist (which has been created in the WM_INITDIALOG handler) needs to be destroyed in a WM_DESTROY handler.

Reply:
This needs to be done too ^

Reply:
No because it's built-in LISTVIEW_NCDestroy. Removing that line is the exact purpose of this PR :-)

Reply:
Heh ok thanks, that's indeed correct and I forgot that.

Reply:
Then it doesn't seem needed to save the imagelist in the window properties:
`SetWindowLongPtrW(hwnd, DWLP_USER, (LPARAM)imageList);` could be removed in the WM_INITDIALOG.

Reply:
If however wine really wants to keep an explicit cleanup, then doing something similar to
https://git.reactos.org/?p=reactos.git;a=blob;f=base/shell/progman/group.c;hb=da36f799c728bc5477e85da590c31e6c20d553c4#l199
should be OK.

Reply:
This Ptr is used elsewhere in the code to recover the cert data structure. I keep it :-)